packages/core/src/App/Containers/Layout/header/signup-onboarding-header.jsx

```diff
@@ -0,0 +1,22 @@
+import classNames from 'classnames';
+import React from 'react';
+import { DesktopWrapper, MobileWrapper, StaticUrl } from '@deriv/components';
+import Logo from 'Assets/SvgComponents/header/deriv-onboarding-logo.svg';
```

Comment:
why are we importing svg directly, can we have it in `components` package and use the `Icon` component?

Reply:
That doesn't support gradient and masks